tests/rkt_net_test.go

```diff
@@ -608,6 +608,10 @@ func testNetCustomNatConnectivity(t *testing.T, nt networkTemplateT) {
 
 	// Child connects to host
 	hostname, err := os.Hostname()
+	if err != nil {
+		t.Fatalf("os.Hostname() got error %q", err)
```

Comment:
Ordinarily I'd say this should be a panic - but is there cleanup we need to perform?